src/xrCore/_math.cpp

```diff
@@ -236,7 +237,11 @@ void _initialize_cpu()
     listFeature("AltiVec", SDL_HasAltiVec());
 
     Msg("* CPU features: %s", features);
+#if defined(XR_PLATFORM_WINDOWS)
+    Msg("* CPU threads: %d", NUMAHardwareConcurrency());
```

Comment:
It's better to just name this function something like `GetThreadsCounts` and place it into CPU namespace in `_math.h` and use it instead of `std::thread::hardware_concurrency());` – that means we also should create a Linux, etc. version of this function.

Reply:
I wish people were saying "physical/logical cores", not "threads". It's so confusing and grows people illiterate IMHO.